lib/package.py

```diff
@@ -30,8 +30,7 @@
 CONF = config.get_config().CONF
 LOG = logging.getLogger(__name__)
 
-# TODO: make this configurable by a base dir parameter
-BUILD_CACHE_DIR = "cache"
+BUILD_CACHE_DIR = os.path.join(CONF.get('default').get('work_dir'), "cache")
```

Comment:
Either mention the build cache directory in the commit message or separate this commit.